src/main/java/frc/robot/commands/Arm/SwingBackServo.java

```diff
@@ -3,47 +3,39 @@
 import edu.wpi.first.wpilibj2.command.Command;
 
 
-public class SwingBack extends Command{
-
+public class SwingBackServo extends Command{
+  
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private final ArmSubsystem armSubsystem;
-  private double velocity;
-  private double acceleration;
-  private double motorAng;
-  private double tolerance;
-  
-
     // Called when the command is initially scheduled.
     
-  public SwingBack( ArmSubsystem armSubsystem, double velocity, 
-  double acceleration, double tolerance) {
+
+  public SwingBackServo(ArmSubsystem armSubsystem)
+  {
     this.armSubsystem = armSubsystem;
-    this.velocity = velocity;
-    this.acceleration = acceleration;
-    this.tolerance = tolerance;
     addRequirements(armSubsystem);
   }
-  public void initialize() {
-    armSubsystem.setAngle(0, velocity, acceleration);
-
+  public void initialize() {   
+    armSubsystem.setServoAngle(0);
   }
+  
 
   // Called every time the scheduler runs while the command is scheduled.
+
   @Override
   public void execute() {
-
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
-    armSubsystem.stopArmMotor();
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    motorAng = armSubsystem.encoderGetAngle();
-    return Math.abs(motorAng) < tolerance;
-  }
-}
+    double motorAng = armSubsystem.getServoAngle();
+  
+    return Math.abs(motorAng) < .001;
```

Comment:
I suspect this tolerance is too small, and might not end the command in a reasonable time.